styleguide/webpack.config.js

```diff
@@ -22,7 +23,45 @@ module.exports = {
       },
       {
         test: /\.svg$/,
-        use: 'file-loader',
+        include: [
+          /src\/svgs\//,
+        ],
+        use: [
+          {
+            loader: 'svg-sprite-loader',
+          },
+          {
+            loader: 'svgo-loader',
+            options: {
+              plugins: [
+                { removeTitle: true },
+                { removeNonInheritableGroupAttrs: true },
+                { removeUselessStrokeAndFill: false },
+                { collapseGroups: true },
+                { convertColors: { shorthex: false } },
+                { convertPathData: false },
+                {
+                  removeAttrs: {
+                    attrs: '(fill|stroke)',
+                    preserveCurrentColor: true,
```

Comment:
These are copied over from what we have in the main application, but without `preserveCurrentColor` for example, our `color: val` based SVGs won't work.

Reply:
What's the reasoning for not bringing that over? Is that not needed for the current SVG in this repo?

Reply:
>What's the reasoning for not bringing that over? Is that not needed for the current SVG in this repo?

I did bring it over and I'm setting it true here. Not sure the confusion.

Reply:
Are you talking about the specific branch for svgo? If so, they merged my PR a long time ago and it was never deemed a priority in bigmaven to point back to `#master` for svgo, but it's in the version we're pulling in here. Fwiw, the alert icon itself uses this so it wouldn't be rendering the caution red if it wasn't being applied ;)